### ISSUE TITLE
If multiple elements have a firebase-query and they all point to the same path, data only loads for one

### ISSUE THREAD

**Issue by miztroh** [firebase/polymerfire#105]
In this example, I'm loading my Firebase app info via ajax (webSetup.json).  I've defined a test element and I create two instances of it.  After the page loads, only one instance actually populates with data.  I experience the same issue even if I have a single instance of two different elements when their ``firebase-query`` instance points to the same path.

```
<!DOCTYPE html>
<html>
	<head>
		<meta charset="utf-8">
		<meta name="viewport" content="width=device-width, initial-scale=1.0">
		<title>Loading...</title>
		<style>
			body {
				margin: 0;
			}
		</style>
		<script src="/bower_components/webcomponentsjs/webcomponents-lite.js"></script>
		<link rel="import" href="/bower_components/polymer/polymer.html">
		<link rel="import" href="/bower_components/iron-ajax/iron-ajax.html">
		<link rel="import" href="/bower_components/polymerfire/firebase-app.html">
		<link rel="import" href="/bower_components/polymerfire/firebase-query.html">
	</head>
	<body>
		<dom-module id="fb-test">
			<template>
				<firebase-query app-name="pp-fb" path="/pages/layouts" data="{{layouts}}" order-by-child="name" id="layouts"></firebase-query>
			</template>
			<script>
				Polymer(
					{
						is: 'fb-test',
						properties: {
							layouts: {
								type: Array
							}
						}
					}
				);
			</script>
		</dom-module>
		<template is="dom-bind">
			<iron-ajax url="/webSetup.json" last-response="{{webSetup}}" auto></iron-ajax>
			<template is="dom-if" if="[[webSetup]]">
				<firebase-app name="pp-fb" api-key="[[webSetup.apiKey]]" auth-domain="[[webSetup.authDomain]]" database-url="[[webSetup.databaseURL]]" storage-bucket="[[webSetup.storageBucket]]"></firebase-app>
				<fb-test id="a"></fb-test>
				<fb-test id="b"></fb-test>
			</template>
		</template>
	</body>
</html>
```

![image](https://cloud.githubusercontent.com/assets/2515209/18020190/d23b37ee-6ba5-11e6-8a86-10f5038d8f7c.png)

**Comment by sruditsky** [comment]
I believe I am experiencing a variation of this problem. 

I have two polymer elements (let's say A and B), each containing several firebase-query elements which are dynamically instantiated (inside <template is="dom-repeat">).

When it happens that a firebase-query in A is the same query as a firebase-query in B, one of the two outcomes are possible (probably depending on the instantiation timing):
1) firebase-query in A loads no data, while firebase-query in B loads data correctly -- (i.e. the same as described by the issue title).
2) firebase-query in A loads data correctly, while firebase-query in B loads data twice.

**Comment by miztroh** [comment]
Can confirm that this issue is resolved by #110.
